blazor/datagrid/excel-exporting.md

```diff
@@ -577,6 +577,68 @@ You can assign the file name for the exported document by defining **fileName**
 }
 ```
 
+### Add additional worksheets to excel file while exporting
+
+Additional worksheets can be added to the excel file while exporting using the [ExcelExportProperties](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.ExcelExportProperties.html) properties. 
```

Comment:
Additional worksheets can be added to the excel file while exporting using the [ExcelExportProperties](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.ExcelExportProperties.html).